editor/assets/stylesheets/main.scss

```diff
@@ -1,7 +1,10 @@
 @import '../../inserter/style';
 
-.gutenberg {
+body {
```

Comment:
We may need to consider a better way to scope this if the styles are to be integrated amongst others in a core admin stylesheet, but this may be fine for now.

Perhaps `body.toplevel_page_gutenberg` ? Or otherwise creating an editor canvas element that's positioned absolute or fixed to the corners of the screen.